### ISSUE TITLE
Support outputting markdown files in folders organised by notebook stack

### ISSUE THREAD

**Issue by loafingaround** [akosbalasko/yarle#353]
Notes in Evernote can be organised into notebook "[stacks](https://help.evernote.com/hc/en-us/articles/208314158)" (groups of notebooks).

If you organise your .enex export files (which each map to a notebook in Evernote) into folders corresponding to your notebook stacks in Evernote (where the notebook in Evernote was in a stack at all of course) it would be good if the output folders (which similarly each map to a notebook) could also be put in folders corresponding to your notebook stacks in Evernote.

**TL;DR** It would be good have the option to reflect notebook stack groupings in the output.

Perhaps this feature could be introduced with a flag in the configuration to switch the behaviour on or off, which maybe defaults to off.

I have actually made some changes to the code to do this for my own purposes so would be happy to contribute this, although it will need some extra work:
* Introducing the on/off configuration flag (described above), if desired
* Integrating the feature (or at least the on/off flag for it) into the GUI client, if desired
* Making the feature work with internal, or "inter-notebook", links - those starting with "evernote://" - as dealt with by https://github.com/akosbalasko/yarle/issues/292
* Unit tests

---

As a side note, as someone using Linux, it would be nice to have an npm script for building on Linux. Again, I've added one for my own purposes - it's just a case of supporting Linux, instead of Windows, path separators: `"build-linux": "tsc --project ./src"`.

**Comment by akosbalasko** [comment]
Hi @loafingaround !

thank you for raising this up, it absolutely makes sense. Please open a PR with your changes and I'll improve it with the rest you mentioned.

Thanks for your contribution!
akos

**Comment by loafingaround** [comment]
Hi, @akosbalasko, 

No problem.

I would be happy to do:
* the on/off configuration flag
* unit tests

although I can't guarantee when I'll get time to do it.

I could probably do:
* the GUI client integration
* making the feature work internal links
but they might take me a bit longer / I might need a bit of help with them.

**Comment by acook** [comment]
@loafingaround did your old code ever end up in a PR?

**Comment by akosbalasko** [comment]
@acook , nope: https://github.com/akosbalasko/yarle/pulls?q=is%3Apr+sort%3Aupdated-desc+author%3Aloafingaround+
